Gemfile

```diff
@@ -1,10 +1,22 @@
 source 'https://rubygems.org'
-ruby '2.4.0'
 
-group :test do
-  gem 'rspec'
-  gem 'cucumber'
-  gem 'rake'
-  gem 'coveralls', require: false
-end
+ruby '2.3.3'
 
+gem 'data_mapper'                 # ORM for sinatra
+gem 'dm-postgres-adapter'         # database
+gem 'selenium-webdriver'
+gem 'shotgun'
+gem 'sinatra'                     # web framework
+gem 'pg'                          # deploying to heroku
+gem 'bcrypt'                      # encrypting passwords for database
+gem 'sinatra-flash'               # displaying error messages
+gem 'rspec'                       # unit testing library
+gem 'rspec-sinatra'               # addon unit testing library for sinatraß
+gem 'capybara'                    # feature testing for web applications
+gem 'cucumber'
+gem 'rake'
+gem 'coveralls', require: false   # analysing test coverage
+gem 'rubocop-rspec'               # analysing test cleanliness/style
+gem 'rubocop'                     # analysing code cleanliness/style
+gem 'database_cleaner'
+gem 'timecop'
```

Comment:
Used the timecop gem for stubbing time in Capybara. Do you know if this is the best way to go about this?